brx-spartacus-library/angular.json

```diff
@@ -12,7 +12,7 @@
         "build": {
           "builder": "@angular-devkit/build-angular:ng-packagr",
           "options": {
-            "project": "ng-package.json"
+            "project": "ng-package.json",
```

Comment:
There is a chance that this will give errors as officially JSON does not have trailing comma's for the last item / key.